src/main/java/net/minecraftforge/common/ForgeTagHandler.java

```diff
@@ -289,21 +282,6 @@ public static void populateTagCollectionManager()
         }
     }
 
-    /**
-     * Updates the custom tag types' tags from reloading via NetworkTagManager
-     *
-     * @apiNote Internal: For use by NetworkTagManager
-     */
-    public static void updateCustomTagTypes(List<TagCollectionReaderInfo> tagCollectionReaders)
-    {
-        ImmutableMap.Builder<ResourceLocation, TagCollection<?>> builder = ImmutableMap.builder();
-        for (TagCollectionReaderInfo info : tagCollectionReaders)
-        {
-            builder.put(info.tagType, info.reader.build(info.tagBuilders));
-        }
-        customTagTypes = builder.build();
```

Comment:
I haven't looked in great detail yet how tags work/changed in 1.17 but it seems like `ForgeTagHandler.customTagTypes` may not being updated anymore post reload due to the removal of this line?